src/epgsqla.erl

```diff
@@ -55,9 +56,13 @@ squery(C, Sql) ->
 equery(C, Sql) ->
     equery(C, Sql, []).
 
--spec equery(epgsql:connection(), #statement{}, [epgsql:bind_param()]) -> reference().
-equery(C, Statement, Parameters) ->
-    cast(C, {equery, Statement, Parameters}).
+-spec equery(epgsql:connection(), #statement{}, [epgsql:typed_param()]) -> reference().
+equery(C, Statement, TypedParameters) ->
+    cast(C, {equery, Statement, TypedParameters}).
```

Comment:
maybe we should move lists:zip inside this function? Or this will break backward compatibility?